pkg/atomicbitops/atomicbitops_benchmark_test.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package atomicbitops
+
+import (
+	"testing"
+)
+
+func BenchmarkAndUint32(b *testing.B) {
+        var x [128]uint32
+        for i := 0; i < b.N; i++ {
+                AndUint32(&x[63], uint32(i))
+        }
+}
+
+func BenchmarkAndUint64(b *testing.B) {
+        var x [64]uint64
+        for i := 0; i < b.N; i++ {
+                AndUint64(&x[31], uint64(i))
```

Comment:
pkg/atomicbitops/atomicbitops_benchmark_test.go:31:13: cannot use &x[31] (type *uint64) as type *Uint64 in argument to AndUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:51:14: cannot use &x[31] (type *uint64) as type *Uint64 in argument to AndUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:67:12: cannot use &x[31] (type *uint64) as type *Uint64 in argument to OrUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:87:13: cannot use &x[31] (type *uint64) as type *Uint64 in argument to OrUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:103:13: cannot use &x[31] (type *uint64) as type *Uint64 in argument to XorUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:123:14: cannot use &x[31] (type *uint64) as type *Uint64 in argument to XorUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:147:24: cannot use ptr (type *uint64) as type *Uint64 in argument to CompareAndSwapUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:148:24: cannot use ptr (type *uint64) as type *Uint64 in argument to CompareAndSwapUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:171:24: cannot use ptr (type *uint64) as type *Uint64 in argument to CompareAndSwapUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:172:24: cannot use ptr (type *uint64) as type *Uint64 in argument to CompareAndSwapUint64
pkg/atomicbitops/atomicbitops_benchmark_test.go:172:24: too many errors